components/mpas-albany-landice/src/shared/mpas_li_time_average_coupled.F

```diff
@@ -0,0 +1,137 @@
+        call mpas_pool_get_array(timeAveragingPool, 'avgBareIceAblation', avgBareIceAblation)
+        call mpas_pool_get_array(timeAveragingPool, 'avgCalvingThickness', avgCalvingThickness)
+        call mpas_pool_get_array(timeAveragingPool, 'avgFaceMeltingThickness', avgFaceMeltingThickness)
+
+        do iCell = 1, nCells
+           avgBareIceAblation(iCell) = 0.0_RKIND
+           avgCalvingThickness(iCell) = 0.0_RKIND
+           avgFaceMeltingThickness(iCell) = 0.0_RKIND
+        end do
+
+        timeAccumulatedCoupled = 0.0_RKIND
+
+    end subroutine li_time_average_coupled_init
+
+!***********************************************************************
+!
+!  routine li_time_average_coupled_accumulate
+!
+!> \brief   Coupled time averager accumulation
+!> \author  Stephen Price
+!> \date    04 March 2025
+!> \details
+!>  This routine accumulated the coupled time averaging fields
+!
+!-----------------------------------------------------------------------
+    subroutine li_time_average_coupled_accumulate(timeAveragingPool, geometryPool, meshPool)
+            
+        use li_setup 
+
+        type (mpas_pool_type), intent(inout) :: timeAveragingPool
+        type (mpas_pool_type), intent(in) :: geometryPool
+        type (mpas_pool_type), intent(in) :: meshPool
+
+        real (kind=RKIND), dimension(:), pointer :: bareIceAblation, avgBareIceAblation, &
+                                                    calvingThickness, avgCalvingThickness, &
+                                                    faceMeltingThickness, avgFaceMeltingThickness
+        integer, pointer :: nCells
+
+        real (kind=RKIND), pointer :: timeAccumulatedCoupled, rhoi, deltat
+
+        integer :: iCell
+
+        call mpas_pool_get_dimension(meshPool, 'nCells', nCells)
+        call mpas_pool_get_config(liConfigs, 'config_ice_density', rhoi)
+        call mpas_pool_get_array(meshPool, 'deltat', deltat)
+        call mpas_pool_get_array(timeAveragingPool, 'timeAccumulatedCoupled', timeAccumulatedCoupled)
+        call mpas_pool_get_array(geometryPool, 'bareIceAblation', bareIceAblation)
+        call mpas_pool_get_array(geometryPool, 'calvingThickness', calvingThickness)
+        call mpas_pool_get_array(geometryPool, 'faceMeltingThickness', faceMeltingThickness)
+        call mpas_pool_get_array(timeAveragingPool, 'avgBareIceAblation', avgBareIceAblation)
+        call mpas_pool_get_array(timeAveragingPool, 'avgCalvingThickness', avgCalvingThickness)
+        call mpas_pool_get_array(timeAveragingPool, 'avgFaceMeltingThickness', avgFaceMeltingThickness)
+
+        do iCell = 1, nCells
+
+           avgBareIceAblation(iCell) = ( avgBareIceAblation(iCell) * timeAccumulatedCoupled &
+                                               + bareIceAblation(iCell) * deltat ) / ( timeAccumulatedCoupled + deltat )
+
+           avgCalvingThickness(iCell) = ( avgCalvingThickness(iCell) * timeAccumulatedCoupled &
```

Comment:
It looks like the units of `avgCalvingThickness` and `avgFaceMeltingThickness` are kg/m^2/s. Can we rename these variables to have more accurate names?

Reply:
Happy to do whatever you and Matt think is best here.

Reply:
I agree with Trevor's suggestion that we rename these variables to better describe what they are actually calculating as opposed to trying to keep their names analogous to their associated prognostic fields.  What do you both think of `avgCalvingFlux` and `avgFacemeltFlux`?

Reply:
Corrected with https://github.com/E3SM-Project/E3SM/pull/7210/commits/4ae313ea218232fca2061d939c9b36ef023227b3.